components/table/ProjectTable.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Box, Heading } from '@chakra-ui/layout';
 import { Table, TableCaption, Thead, Tr, Td, Th, Tbody } from '@chakra-ui/react';
-import { components } from '@/lib/types/api';
 import Link from 'next/link';
+import { ProjectDTO } from '@/lib/types/dto';
 
 type ProjectListProps = {
-    projects: components['schemas']['ProjectDTO'][];
```

Comment:
There does not seem to be a good reason to prefer `components['schemas']['ProjectDTO'][]` over `ProjectDTO[]` with an exported conversion somewhere